### DISCUSSION TITLE
중고거래 페이지에서 판매할 책을 등록할 때 방식

### DISCUSSION THREAD

**Discussion by JangGusWjd** [MajorIn-ABO/MajorIn_FE#18] [Ideas]
중고거래 페이지에서 책을 판매할 때
1. 책 제목, 저자, 출판사, 판매금액, 교재 표지를 사용자에게 입력받아 등록하는 방법(그때그때 입력 폼처럼)
2. 책 제목, 저자 검색시 필터링 된 관련 책 리스트들이 다 나와서 거기서 선택 후 판매 금액만 입력하는 방법(책이 리스트에 없는 경우 수동으로 등록되어 그때 리스트에 추가)

**Comment by cho-ja-young** [comment]
오 원래 나는 1번을 생각하기는 했는데, 2번을 도서 검색 API 가져와서 사용하면 괜찮을듯?? 
-> 원래 가격 정보도 있으면 가져와서 보여주는 것도 좋을 것 같고! 얼마나 싸게 파는지 비교를 위해서하던지 
요거는 도서정보 API 예시 ! ( https://myunji.tistory.com/472 )

**근데 대신 정보를 끌어오더라도 책 표지나 내부 등 책 상태를 보여줄 수 있는 사진 첨부 필요성은 있는 것 같음!!**

**Comment by wlsl10** [comment]
웅 나도 책을 판매해볼 때 보니 책 사진 미리 찍어두고 밖에서 판매 글 올릴 때도 있었어서 2번으로 하면 책이 바로 없어도 글을 올릴 수 있어서 편할 것 같아!
